### PR TITLE
[A11Y-77] Remove __useDeprecatedTag usage from TeacherDashboardHeader.jsx

### PR DESCRIPTION
**Pull request by levadadenys**
### [A11Y-77] Remove __useDeprecatedTag usage from TeacherDashboardHeader.jsx

Before:
Знімок екрана 2023-04-12 о 16 47 15

After:
Знімок екрана 2023-04-12 о 17 00 42

## Links
- jira ticket: [[A11Y-77]](https://codedotorg.atlassian.net/browse/A11Y-77?atlOrigin=eyJpIjoiOWZhMjkyYzE0ZjY5NDE0MGFlYmZjMWRjNjI5NTczZmIiLCJwIjoiaiJ9)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/teacherDashboard/TeacherDashboardHeader.jsx [hunk 1]
```diff
@@ -169,7 +168,8 @@ const styles = {
     marginBottom: 5
   },
   buttonWithMargin: {
-    marginRight: '5px'
+    margin: 0,
```
Comment (molly-moen) [inline apps/src/templates/teacherDashboard/TeacherDashboardHeader.jsx:171]: what's the reason for this change?

Comment (levadadenys) [reply apps/src/templates/teacherDashboard/TeacherDashboardHeader.jsx:171]: Glad you asked! Since it'll be `button` instead of a `div` it'll have `margin: 5px; `style from `application.scss` button base style. This line is needed to overwrite base style and keep the button appearance be the same it was before changing `div` to `button`

**Review by molly-moen — APPROVED** [review]
